constance/backends/database/__init__.py

```diff
@@ -87,7 +86,7 @@ def set(self, key, value):
         if self._cache:
             self._cache.set(key, value)
 
-        updated_signal.send(sender='constance', key=key, value=value)
+        signals.config_updated.send(sender='constance', key=key, value=value)
```

Comment:
I'll be hard to get the **old** value as it will add another hit to the backend. I believe for now only getting notified about the change is best.